common/services/prismic/exhibitions.js

```diff
@@ -335,9 +333,9 @@ function putPermanentAfterCurrentExhibitions(
   ];
 }
 
-export async function getExhibition(
-  req: ?Request,
+async function getExhibition(
   id: string,
+  req: ?Request,
```

Comment:
This is prep for a TypeScript lint that says required params can't come after optional ones, but I wasn't able to get exhibitions.js actually into TypeScript.